app/src/main/java/com/internship/firstweekapp/dict/Dictionary.kt

```diff
@@ -0,0 +1,116 @@
+package com.internship.firstweekapp.dict
+
+import android.content.Context
+import com.internship.firstweekapp.R
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+
+enum class State {
+    PL_UK, UK_PL
+}
+
+class Dictionary(var context: Context) {
+    private val wordMapPL = mutableMapOf<String, ArrayList<String>>()
+    private val wordMapUA = mutableMapOf<String, ArrayList<String>>()
+    private var state = State.PL_UK
+
+    suspend fun read() {
```

Comment:
remove duplicated withContext(Dispatchers.IO) and make method from inside code block